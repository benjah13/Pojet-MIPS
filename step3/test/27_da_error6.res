00000000:	NOP
00000004:	NOP
DA function only takes one argument! Format da 0xaddr:nb_instr
