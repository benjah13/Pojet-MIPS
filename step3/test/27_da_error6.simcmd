# test da erreur pas de :
da 0xae1:2
da 0xae1f:100 am 
