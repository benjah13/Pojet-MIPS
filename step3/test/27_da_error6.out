00000ae0:	NOP
00000ae4:	NOP
DA function only takes one argument! Format da 0xaddr:nb_instr
